### ISSUE TITLE
Terrapin

### ISSUE THREAD

**Issue by dhwalker** [connectbot/sshlib#432]
Is ConnectBot vulnerable to to Terrapin, and if so, will a fix be forthcoming?

**Comment by dhwalker** [comment]
It looks like the recent release resolved the the Terrapin vulnerabilities. Thanks!

**Comment by kruton** [comment]
Yes, it was fixed with commit 5c8b534f6e97db7ac0e0e579331213aa25c173ab.
